lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/services/UserAccessServiceImpl.java

```diff
@@ -30,11 +30,11 @@ public UserAccessServiceImpl() {
     public Optional<UserData> login(String sessionId, LoginRequest loginRequest) {
         UserData userData = users.get(loginRequest.getUserName());
         if (userData != null && userData.verifyPassword(loginRequest.getPassword())) {
-            LOG.info("login OK: {} {}", sessionId, loginRequest.getUserName());
+            LOG.info("login OK: {}", sessionId);
```

Comment:
the username is also helpful in logs for this example

Reply:
done